docs/tutorials/setting-up-an-nfs-fileShare.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Setting Up an NFS Fileshare
+subject: Networking
+author: Anton Vanhoucke, Ralph Hempel
+---
+
+* Table of Contents
+{:toc}
+
+## Why Do It?
+
+The nfs protocol is the standard Linux way to share files between computers, and since you're running Linux on the EV3, and there's nfs support built-in, you can share files between your computer and the EV3.
+
+Using a file share method like nfs makes it easier for folks like me to load and unload modules that are under development. That saves time and is less error prone than swapping SD Cards and rebooting all the time.
+
+For general users, it makes it easy to upgrade their SD Cards with known good modules, new kernel `uImage` files, and other updates to config files. It also makes it easier to backup your SD card. Finally, it allows you to edit files on your host machine, then run them on the EV3 without having to actually copy files anywhere!
+
+![nfs_on_ev3dev](Images/Website/nfs_on_ev3dev.png)
+
+It's relatively easy, you just need to keep the addresses and filenames straight. All you have to do is modify one file on the EV3 and one on your host computer. Let's start with getting the computer set up.
+
+The instructions here are for all three platforms Linux, Windows, and OSX - just pick the one you need 
+
+## How To Do It - Linux
+
+On your Linux box, you'll need to edit a file called `/etc/exports`. If you don't have this file, then you need to install `nfs-common` or a similar package. Update the file to look like this:
+
+```
+# /etc/exports: the access control list for filesystems which may be exported
```

Comment:
Assuming that these comments show up in this file by default, I think it would be clearer if the comments were omitted and it just said "... Open the file and add the following line:".